back/engines/commercial/analysis/app/services/analysis/side_fx_analysis_service.rb

```diff
@@ -30,5 +34,14 @@ def create_example_tags(analysis)
         tag_type: 'onboarding_example'
       )
     end
+
+    def fallback_custom_fields(analysis)
```

Comment:
@kogre Perhaps it could be good to double check if this still behaves as expected.

Reply:
I'm sorry, I had to edit this because there were reloading issues 😅